pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -335,6 +366,13 @@ func LoadIPSecKeysFile(path string) (int, uint8, error) {
 	return loadIPSecKeys(file)
 }
 
+// LoadIPSecKeys imports IPSec auth and crypt keys from a byte[]. The format
+// is a key per line as follows, ([id] auth-algo auth-key enc-algo enc-key)
+// Returns the authentication overhead in bytes, the key ID, and an error.
+func LoadIPSecKeys(keys []byte) (int, uint8, error) {
```

Comment:
@jrfastab it seems this function was only added for unit tests. Why not having it in the unit tests files?

Reply:
Right, sure we can add it to unit test files instead.